Content.Shared/Construction/Prototypes/ConstructionPrototype.cs

```diff
@@ -69,6 +75,12 @@ public sealed class ConstructionPrototype : IPrototype
         [DataField("canRotate")]
         public bool CanRotate { get; } = true;
 
+        /// <summary>
+        ///     Construction to replace this construction with when the current one is 'flipped'
+        /// </summary>
+        [DataField("mirror", customTypeSerializer:typeof(PrototypeIdSerializer<ConstructionPrototype>))]
+        public string Mirror { get; } = string.Empty;
```

Comment:
This should just be a construction step.

Reply:
Hi, An important part of the feature is to have feedback. If I flipped the ghost, I want to be able to see it. So I need to store a sprite. If I store a sprite, I think the simplest solution is to use another `ConstructionPrototype`. I was inspired by the `FlippableComponent`. Because I need another `ConstructionPrototype` the `Hide` field is needed.

If I missed your comment, can you explain in depth what you think ?

Reply:
I have pushed this pr https://github.com/space-wizards/space-station-14/pull/14492. It adds a new field `infereSprite` which make the proposition of using a construction step possible.

I was thinking of replacing `TargetNode` by `TargetNodes`. It will enable the possibility to cycle between multiple target node. The problem is this modification isn't retro-compatible. And it will break a lot of construction prototype definition.

I could create a custom serializer that can take ValueDataNode or a SequenceDataNode to build the `TargetNode`. It will be retro-compatible, but it is less logical.

What do you think ?

Reply:
I was hoping vera would get a chance to look but she's pretty retired so I'm going to merge as is and if we need to can make adjustments after.